### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (span:nth-child(1))

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#65]
#### Issue
![image](https://user-images.githubusercontent.com/1296369/69329114-5dda7d00-0c48-11ea-9413-04406a986ed6.png)

`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI))
#### Target application
[Modules (Site) - kh - Administration](http://localhost/kh/administrator/index.php?option=com_modules&view=modules&client_id=0)
#### Element path
#sorted > span:nth-child(1)
#### Snippet
`<span>
			Title		</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.19 (foreground color: #0184ff, background color: #edf0f5, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 78.0.3904.97
